src/outputChannel.ts

```diff
@@ -29,7 +29,18 @@ export class D2OutputChannel {
       String(this.pad(now.getSeconds())),
     ];
 
-    this.outputChannel.appendLine(`[${time.join(":")}] - ${msg}`);
+    const msgs: string[] = msg.split("\n");
+
+    for (const m in msgs) {
+      if (msgs[m]) {
+        const output = msgs[m];
+        if (output.length === 0) {
+          continue;
+        }
+        this.outputChannel.appendLine(`[${time.join(":")}] - ${output}`);
+      }
+    }
```

Comment:
```suggestion
    for (const m of msgs) {
      if (m.length === 0) {
        continue;
      }
      this.outputChannel.appendLine(`[${time.join(":")}] - ${m}`);
    }
```

Reply:
I learn something new everyday... TypeScript is not my first language ;)